<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffInforDialog</class>
 <widget class="QDialog" name="StaffInforDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>542</width>
    <height>368</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>542</width>
    <height>368</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Thông tin nhân viên</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources.qrc">
    <normaloff>:/images/icon/note.png</normaloff>:/images/icon/note.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#StaffInforDialog {
	background-color: #f5f5fa;
}</string>
  </property>
  <widget class="QLabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>542</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24px;
font-weight: bold;
color: #f27d85;</string>
   </property>
   <property name="text">
    <string>Thông tin nhân viên</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="orderDetailClose">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>323</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f2758d;
font-size: 16px;
color: white;
font-weight: bold;
padding: 6px 10px;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Đóng</string>
   </property>
  </widget>
  <widget class="QWidget" name="wrapper" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>57</y>
     <width>511</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	font-size: 16px;
	background-color: white;
	border-radius: 20px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="widget_32" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_25" stretch="0,0,1">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_54">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Tên nhân viên:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="name">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #333;
font-size: 18px;</string>
         </property>
         <property name="text">
          <string>Hoàng Trọng Vũ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="username">
         <property name="text">
          <string>(htvu)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_40" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_35" stretch="0,1">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_71">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Số điện thoại:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="phoneNumber">
         <property name="text">
          <string>12345</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_42" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_36" stretch="0,1">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_72">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Địa chỉ:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="address">
         <property name="text">
          <string>VNVNVNV</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_49" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_43" stretch="0,1">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_73">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Số sản phẩm đã bán được:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="soldQuantity">
         <property name="text">
          <string>20</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_43" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_37" stretch="0,1,0,0">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_76">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Lương cơ bản:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="baseSalary">
         <property name="text">
          <string>100.000đ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_77">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Thưởng thêm:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bonus">
         <property name="text">
          <string>50.000đ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_44" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_38" stretch="0,1">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_78">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #777;</string>
         </property>
         <property name="text">
          <string>Lương tháng này:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalSalary">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
color: #f27d85;
font-size: 18px;</string>
         </property>
         <property name="text">
          <string>150.000đ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
